src/Graze/Monolog/Handler/RaygunHandler.php

```diff
@@ -45,7 +45,7 @@ protected function write(array $record)
 
         if (isset($context['exception']) && $context['exception'] instanceof \Exception) {
             $this->writeException(
-                $record['formatted'],
```

Comment:
The `NormalizerFormatter` converts `Exception`s and `Throwable`s into arrays. The Raygun client needs an actual `Exception` or `Throwable` object in order to format it's message